wrapper/pycc/src/qcc_db/ccPointCloud.cpp

```diff
@@ -83,9 +83,14 @@ void define_ccPointCloud(py::module &m)
         .def("setCurrentDisplayedScalarField",
              &ccPointCloud::setCurrentDisplayedScalarField,
              "index"_a)
+        .def("setPointSize", &ccPointCloud::setPointSize, "size"_a)
         .def("sfColorScaleShown", &ccPointCloud::sfColorScaleShown)
         .def("showSFColorsScale", &ccPointCloud::showSFColorsScale, "state"_a)
         .def("addPoints", &PyCC::addPointsFromArrays<ccPointCloud>)
+        .def("setColor", [](ccPointCloud &self, unsigned char r, unsigned char g, unsigned char b, unsigned char a) {
```

Comment:
instead of `unsigned char` I would use `ColorCompType` as its the type used in the cpp definition